Makefile

```diff
@@ -0,0 +1,104 @@
+# Makefile for the bundlechanges library.
+
+ifndef GOPATH
+$(warning You need to set up a GOPATH.)
```

Comment:
This causes a false warning to be printed when using targets (e.g. sysdeps) that don't actually require GOPATH.  <meh>